pootle/apps/pootle_statistics/models.py

```diff
@@ -49,11 +49,10 @@ class SubmissionTypes(object):
     # Combined types that rely on other types (useful for querying)
     # Please use the `_TYPES` suffix to make it clear they're not core
     # types that are stored in the DB
-    EDIT_TYPES = [NORMAL, SYSTEM]
+    EDIT_TYPES = [NORMAL, SYSTEM, UPLOAD]
     CONTRIBUTION_TYPES = [NORMAL, SYSTEM, SUGG_ADD]
     SUGGESTION_TYPES = [SUGG_ACCEPT, SUGG_ADD, SUGG_REJECT]
     REVIEW_TYPES = [SUGG_ACCEPT, SUGG_REJECT]
-    EDITING_TYPES = [NORMAL, SYSTEM, UNIT_CREATE, UPLOAD]
```

Comment:
@julen, we discussed with @phlax and @dwaynebailey that `Submission.type` should be split for  `Submission.source` and `Submission.action`. That's related to our approach with s/Submission/Event/ in EN. As I understood we won't address it in this issue.